tensorflow/python/ops/ragged/ragged_array_ops.py

```diff
@@ -871,6 +871,66 @@ def _cross_internal(inputs,
     return ragged_tensor.RaggedTensor.from_row_splits(
         values_out, splits_out, validate=False)
 
+@tf_export("ragged.fill_empty_rows")
+def fill_empty_rows(ragged_input,
+                           default_value,
+                           name=None):
+  """Fills empty rows in the input `RaggedTensor` with rank 2 with a default
+  value.
+
+  This op adds entries with the specified `default_value` for any row in the
+  input that does not already have a value.
+
+  The op also returns an indicator vector such that
+
+      empty_row_indicator[i] = True iff row i was an empty row.
+
+  Args:
+    ragged_input: A `RaggedTensor` with rank 2.
+    default_value: The value to fill for empty rows, with the same type as
+      `ragged_input.`
+    name: A name prefix for the returned tensors (optional)
+
+  Returns:
+    ragged_ordered_output: A `RaggedTensor`with all empty rows filled in with
+      `default_value`.
+    empty_row_indicator: A bool vector indicating whether each input row was
+      empty.
+
+  Raises:
+    TypeError: If `ragged_input` is not a `RaggedTensor`.
```

Comment:
Should we raise an exception if the rank is not 2?

Reply:
See the comment above on verifying the rank.